src/app/index.ts

```diff
@@ -276,42 +243,54 @@ module.exports = yo.extend({
   writing: {
     copyFiles: function () {
       try {
-        let language = this.project.ts ? 'ts' : 'js';
+        let language = this.project.scriptType  === typescript ? 'ts' : 'js';
 
         /** Show type of project creating in progress */
-        if (this.project.framework !== 'manifest-only') {
+        if (!this.project.isManifestOnly && !this.project.isExcelFunctionsProject) {
           this.log('\n----------------------------------------------------------------------------------\n');
-          this.log(`      Creating ${chalk.bold.green(this.project.projectDisplayName)} add-in using ${chalk.bold.magenta(language)} and ${chalk.bold.cyan(this.project.framework)}\n`);
+          this.log(`      Creating ${chalk.bold.green(this.project.projectDisplayName)} add-in at ${chalk.bold.magenta(this._destinationRoot)} for ${chalk.bold.yellow(this.project.host)} using ${chalk.bold.magenta(language)}\n`);
           this.log('----------------------------------------------------------------------------------\n\n');
         }
-        else {
-          this.log('----------------------------------------------------------------------------------\n');
-          this.log(`      Creating manifest for ${chalk.bold.green(this.project.projectDisplayName)} add-in\n`);
+        else if (this.project.isExcelFunctionsProject) {

```

Comment:
Do we want/need a different banner here for an Excel custom functions project?

Reply:
The reason I created a separate banner for custom functions is that custom functions aren't strictly associated with a particular host type (i.e users are not required or prompted for a host type if projectType == excel-functions